catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/CswFilterFactory.java

```diff
@@ -835,4 +837,37 @@ private BinarySpatialOpType createBinarySpatialOpType(PropertyNameType propertyN
         return binarySpatialOpType;
     }
 
+    //default implementation just assume the first argument is a parameter name and the rest are values
+    public FilterType buildPropertyIsEqualTo(String functionName, List<Object> arguments,
+            Object literal) {
+        List<JAXBElement<?>> expressions = new ArrayList<>();
+        for (int i = 0; i < arguments.size(); i++) {
+            if (i == 0) {
+                expressions.add(createPropertyNameType(Arrays.asList(new Object[] {
+                        arguments.get(i)})));
+            } else {
+                expressions.add(createLiteralType(arguments.get(i)));
```

Comment:
may just have this do all literals since they just get turned into objects anyway it doesn't matter which are propertyNames and which are strings the delegate will just assume based on the parameter order